examples/using_fhir_views.ipynb

```diff
@@ -0,0 +1,671 @@
+    "# where this notebook is running.\n",
+    "analysis_dataset = \"statin_analysis_example\"\n",
+    "\n",
+    "dialects.registry.register(\"hive\", \"pyhive.sqlalchemy_hive\", \"HiveDialect\")\n",
+    "\n",
+    "# The endpoint of the Hive ThriftServer to connect to\n",
+    "query_engine = engine.create_engine(\"hive://localhost:10001/default\")\n",
+    "\n",
+    "# Create a runner to execute the views over Spark.\n",
+    "runner = spark_runner.SparkRunner(\n",
+    "    query_engine=query_engine,\n",
+    "    fhir_dataset=fhir_dataset,\n",
+    "    view_dataset=analysis_dataset,\n",
+    "    snake_case_resource_tables=True,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The cell below loads the parquet files we've downloaded into our Spark ThriftServer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "destination_directory_path = \"/dwh/controller_*\"\n",
+    "with query_engine.connect() as curs:\n",
+    "    curs.execute(f\"DROP TABLE IF EXISTS {fhir_dataset}.encounter;\")\n",
+    "    curs.execute(f\"DROP TABLE IF EXISTS {fhir_dataset}.observation;\")\n",
+    "    curs.execute(f\"DROP TABLE IF EXISTS {fhir_dataset}.patient;\")\n",
+    "    curs.execute(\n",
+    "        f\"CREATE TABLE IF NOT EXISTS {fhir_dataset}.encounter USING\"\n",
+    "        f\" PARQUET LOCATION '{destination_directory_path}/Encounter/*.parquet';\"\n",
+    "    )\n",
+    "    curs.execute(\n",
+    "        f\"CREATE TABLE IF NOT EXISTS {fhir_dataset}.observation USING PARQUET\"\n",
+    "        f\" LOCATION '{destination_directory_path}/Observation/*.parquet';\"\n",
+    "    )\n",
+    "    curs.execute(\n",
+    "        f\"CREATE TABLE IF NOT EXISTS {fhir_dataset}.patient USING PARQUET\"\n",
+    "        f\" LOCATION '{destination_directory_path}/Patient/*.parquet';\"\n",
+    "    )\n",
+    "\n",
+    "    curs.execute(f\"DROP DATABASE IF EXISTS {analysis_dataset} CASCADE;\")\n",
+    "    curs.execute(f\"CREATE DATABASE IF NOT EXISTS {analysis_dataset};\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explore the data"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Getting a general feeling for the data is an important first step in any analysis. So let's create some views of FHIR resources and take a look at common fields.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load views based on the base FHIR R4 profile definitions.\n",
+    "views = r4.base_r4()\n",
+    "\n",
+    "# Create shorthand names for resources we will work with.\n",
+    "conds = views.view_of(\"Condition\")\n",
+    "meds = views.view_of(\"MedicationRequest\")\n",
```

Comment:
Hmm, do you need `Condition` and `MedicationRequest`? If yes, they are not included in the default flags, so please make a comment about that. If no, then maybe remove these?

Reply:
that is true! I pasted this from the FHIR Views example and forgot to remove these